src/fe/inf_fe_map.C

```diff
@@ -29,6 +29,18 @@
 namespace libMesh
 {
 
+bool libmesh_isfinite(Real value)
+{
+  // check if \p value is infinite
+  if( value > std::numeric_limits<Real>::max()
+         || value <  std::numeric_limits<Real>::min() )
```

Comment:
I'm not sure if comparisons against `max` are really the right way to go.  At least in this stack overflow [thread](http://stackoverflow.com/questions/2249110/how-do-i-make-a-portable-isnan-isinf-function/2249173) no one is taking that approach, and it seems like those people are knowledgable.  I have a commit based on your 26a697d which I am testing now, will post a PR if it seems to work.